src/Microsoft.Azure.NotificationHubs/Auth/TokenProvider.cs

```diff
@@ -15,9 +17,15 @@ namespace Microsoft.Azure.NotificationHubs.Auth
     /// </summary>
     public abstract class TokenProvider : IDisposable
     {
+        private static readonly TimeSpan DefaultTokenTimeout = TimeSpan.FromMinutes(20.0);
+        private static readonly TimeSpan InitialRetrySleepTime = TimeSpan.FromMilliseconds(50.0);
         private readonly IMemoryCache _tokenCache;
         private readonly bool _cacheTokens;
         private readonly TimeSpan _cacheExpirationTime;
+        private readonly bool _isWebTokenSupported;
+        private readonly TimeSpan _retrySleepTime;
+        private readonly int _cacheSize;
+        private const TokenScope DefaultTokenScope = TokenScope.Entity;
```

Comment:
This doesn't seem to be used, is that intentional?

Reply:
removed